D/DecompUtil/build_tarballs.jl

```diff
@@ -0,0 +1,37 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "DecompUtil"
+version = v"0.1.0"
+
+# Collection of sources required to complete build
+sources = [
+    GitSource("https://github.com/dev10110/DecompUtil_C.git", "982449a84f620488fe9d954eb9a6d301dddfff3f")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/DecompUtil_C
+mkdir build && cd build
+cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release .. 
```

Comment:
```suggestion
if [[ "${target}" == *-mingw* ]]; then
   # Make `M_PI` available
    export CXXFLAGS="-D_USE_MATH_DEFINES"
fi
cmake -DCMAKE_INSTALL_PREFIX=$prefix -DCMAKE_TOOLCHAIN_FILE=${CMAKE_TARGET_TOOLCHAIN} -DCMAKE_BUILD_TYPE=Release .. 
```

Reply:
But since this is your code, you should really do in your package.  We shouldn't have to manually define macros that you don't define.  Something like
```c
#ifdef _WIN32
#define _USE_MATH_DEFINES
#endif
```
_before_ importing `cmath` or `math.h`  would do the job

Reply:
Cool. I made changes in the source directory. 
Thanks for sharing about this.